modules/nextflow/src/main/groovy/nextflow/executor/BashWrapperBuilder.groovy

```diff
@@ -307,6 +307,7 @@ class BashWrapperBuilder {
 
         final binding = new HashMap<String,String>(20)
         binding.header_script = headerScript
+        binding.process_directives = getProcessDirectives()
```

Comment:
I would call this `task_info` or `task_metadata` because directives is also used for the scheduler meta-comments